### ISSUE TITLE
Platypus: Utility panel's "Lesson Notes" dropdown missing accessible name

### ISSUE THREAD

**Issue by eddybrando** [Qiskit/platypus#478]
According to an Axe a11y audit performed on `/course/ch-prerequisites/environment-setup-guide-to-work-with-qiskit-textbook`, the Utility panel's "Lesson Notes" dropdown contains a `listbox` that is missing an accessible name according to WCAG 4.1.2: https://dequeuniversity.com/rules/axe/3.5/aria-input-field-name

This might be tricky to solve as it seems to be an issue with the Carbon dropdown component, which according to its [accessibility section](https://www.carbondesignsystem.com/components/dropdown/accessibility/) has some accessibility issues.

We should still try and see if something can be done from our end to solve this our maybe submit a PR to Carbon to solve this if it's an upstream issue.

---

https://dequeuniversity.com/rules/axe/3.5/aria-input-field-name

---

**User impact:** Serious
**Disabilities affected:** Blind, Deafblind, Mobility
**Requirement(s):** N/A
**WCAG Success Criteria:** 4.1.2 Name, Role, Value

**Comment by vabarbosa** [comment]
This repository is no longer maintained. It has been superseded by [IBM Quantum Learning](https://learning.quantum.ibm.com).
